opentimelineio/plugins/manifest.py

```diff
@@ -143,6 +145,19 @@ def load_manifest():
     except ImportError:
         pass
 
+    # Discover setuptools-based plugins
+    for plugin in pkg_resources.iter_entry_points("opentimelineio.plugins"):
+        plugin_name = plugin.name
+        try:
+            plugin_entry_point = plugin.load()
+            plugin_manifest = plugin_entry_point.plugin_manifest()
+        except Exception:
+            logging.exception("could not load plugin: {}".format(plugin_name))
+            continue
+
+        result.adapters.extend(plugin_manifest.adapters)
+        result.adapters.extend(plugin_manifest.media_linkers)
```

Comment:
Looks like this should be `result.media_linkers.extend`, yah?

Reply:
Good catch, fixed!